python/cuml/test/test_random_forest.py

```diff
@@ -522,6 +522,7 @@ def test_rf_regression_sparse(special_reg, datatype, fil_sparse_format, algo):
             assert fil_r2 >= (sk_r2 - 0.07)
 
 
+@pytest.mark.xfail(reason='Need rapidsai/rmm#415 to detect memleak robustly')
```

Comment:
There are also mem leak tests in svm that would need to be disabled as part of this PR:

https://github.com/rapidsai/cuml/blob/4b62cf654cdfd414539a5b361d4e05edd4cabfb4/python/cuml/test/test_svm.py#L365